service-base/src/main/java/org/eclipse/hono/service/credentials/BaseCredentialsService.java

```diff
@@ -45,165 +32,13 @@
  *
  * @param <T> The type of configuration class this service supports.
  */
-public abstract class BaseCredentialsService<T> extends EventBusService<T> implements CredentialsService {
-
-    private static final String SPAN_NAME_GET_CREDENTIALS = "get Credentials";
-
-    @Override
-    protected String getEventBusAddress() {
-        return CredentialsConstants.EVENT_BUS_ADDRESS_CREDENTIALS_IN;
-    }
+@Deprecated
```

Comment:
I think adding a `@deprecated` JavaDoc indicating what to use instead would be helpful